cves/kernel/CVE-2014-9730.yml

```diff
@@ -55,7 +55,18 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A symlink is kind of like a shortcut in windows. There are some differences, 
+  but they are not important here. In the code that handles these links there is a 
+  part that uses a length inside the symlinks to read from a buffer. There was no 
+  check to make sure the size of the length fit in the buffer. Users without root 
+  privilege could use a corrupted UDF file system image to force the kernel to 
+  crash. UDF is just a type of file system called Universal Disk Format. Using 
+  the correct image an attacker could force the kernel to read passed the buffer 
+  when traversing the symlink causing a crash. This vulnerability is actually a 
+  related vulnerabiltity to this original issue. This vulnerability is speicifically 
+  addressing an issue where when traversing a symlink, some of the components do not 
+  use a length property and it is not ignored for those types of components.
```

Comment:
Good description. Good job defining terms that some people may not be familiar with.